### ISSUE TITLE
Sample app error using Parcel.js

### ISSUE THREAD

**Issue by nevf** [hybridsjs/hybrids#17]
I've ```npm install hybrids``` and copied the files from the ```simple-counter``` example app. 

When I run: ```parcel index.html``` I get:
````
utils.js:50 Uncaught TypeError: Cannot use 'in' operator to search for 'ActiveXObject' in undefined
    at Object.parcelRequire.node_modules/hybrids/esm/utils.js (utils.js:50)
    at newRequire (brutal.e31bb0bc.js:49)
    at localRequire (brutal.e31bb0bc.js:55)
    at Object.parcelRequire.node_modules/hybrids/esm/property.js../utils (property.js:1)
    at newRequire (brutal.e31bb0bc.js:49)
    at localRequire (brutal.e31bb0bc.js:55)
    at Object.parcelRequire.node_modules/hybrids/esm/define.js../property (define.js:1)
    at newRequire (brutal.e31bb0bc.js:49)
    at localRequire (brutal.e31bb0bc.js:55)
    at Object.parcelRequire.node_modules/hybrids/esm/index.js../define (index.js:1)
````
because ```global``` is undefined. If I edit utils.js and change ```global``` to ```window``` the app runs correctly, but this is of course a hack.

[Parcel.js home page.](https://parceljs.org/)

**Comment by smalluban** [comment]
Thank you for pointing this out.  This is obviously an oversight, as the `esm` target is not compiled through the webpack, `global` is not transpiled to `window` object. I will prepare fix and release new patch version.

**Comment by smalluban** [comment]
Fixed in `v1.4.2`.
